.github/workflows/release-docker.yml

```diff
@@ -0,0 +1,45 @@
+name: Publish Docker image
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Docker meta
+        id: docker_meta
+        uses: crazy-max/ghaction-docker-meta@v1
```

Comment:
I don't really understand what this action actually does?

`steps.docker_meta.outputs.tags` will contain `${GITHUB_REF#refs/tags/v}` (for instance: `0.6.0`)?And what about `steps.docker_meta.outputs.labels`?

Reply:
I think we could use something like that:

```yml
- name: Docker metadata
  id: docker_metadata
  run: echo ::set-output name=labels::${IMAGE_NAME}:${ANTORA_VERSION}-lunr${GITHUB_REF/refs\/tags\//},${IMAGE_NAME}:${ANTORA_VERSION},${IMAGE_NAME}:latest
  env:
    IMAGE_NAME: "ggrossetie/antora-lunr"
    ANTORA_VERSION: "2.3.4"
```

(untested)

Reply:
The output of `steps.docker_meta.outputs.labels` is:

```bash
"Labels": {
                "org.opencontainers.image.created": "2021-03-14T15:55:40.915Z",
                "org.opencontainers.image.description": "An Antora's site generator that produces a Lunr index",
                "org.opencontainers.image.licenses": "MIT",
                "org.opencontainers.image.revision": "136a1c3a73d63ea38bd17a3ae363fe46ab387ede",
                "org.opencontainers.image.source": "https://github.com/Adito5393/antora-site-generator-lunr",
                "org.opencontainers.image.title": "antora-site-generator-lunr",
                "org.opencontainers.image.url": "https://github.com/Adito5393/antora-site-generator-lunr",
                "org.opencontainers.image.version": "v100.0.0"
            }
```

Available when the user inspects the image (i.e. `docker image inspect adito5393/antora-v2.3.4-lunr`).

It is up to you to decide if you think this info is redundant! I'm fine with leaving it out. (FYI `docker image inspect antora/antora:latest` contains no labels info).

Reply:
> I think we could use something like that:
> 
> ```yaml
> - name: Docker metadata
>   id: docker_metadata
>   run: echo ::set-output name=labels::${IMAGE_NAME}:${ANTORA_VERSION}-lunr${GITHUB_REF/refs\/tags\//},${IMAGE_NAME}:${ANTORA_VERSION},${IMAGE_NAME}:latest
>   env:
>     IMAGE_NAME: "ggrossetie/antora-lunr"
>     ANTORA_VERSION: "2.3.4"
> ```
> 
> (untested)

I have implemented the versioning you proposed, check [this repo](https://hub.docker.com/r/adito5393/antora-lunr) to see the final result of the `release-docker.yml`.

You can see [here](https://github.com/Adito5393/antora-site-generator-lunr/actions), that I triggered the action 2 times with the different tags.

Reply:
> Available when the user inspects the image

Oh I see, that's quite nice, let's keep it 😉 

> I have implemented the versioning you proposed, check this repo to see the final result of the release-docker.yml.

It looks good 👏🏻